src/gui/optionsdialog.ui

```diff
@@ -1830,6 +1830,19 @@ readme[0-9].txt: filter 'readme1.txt', 'readme2.txt' but not 'readme10.txt'.</st
                  </item>
                 </layout>
                </item>
+               <item>
+                <widget class="QCheckBox" name="checkProxyHostnames">
```

Comment:
Please move this checkbox lower than "Use proxy only for torrent" option.

And adjust code relative position (in every source code files you modified) to match this.

Reply:
Done.